relayer/message_relayer.go

```diff
@@ -311,7 +411,7 @@ func (r *messageRelayer) createSignedMessage(requestID uint32) (*warp.Message, e
 					}
 
 					// As soon as the signatures exceed the stake weight threshold we try to aggregate and send the transaction.
-					if utils.CheckStakeWeightExceedsThreshold(accumulatedSignatureWeight, totalValidatorWeight, utils.DefaultQuorumNumerator, utils.DefaultQuorumDenominator) {
+					if utils.CheckStakeWeightExceedsThreshold(accumulatedSignatureWeight, totalValidatorWeight, params.WarpDefaultQuorumNumerator, params.WarpQuorumDenominator) {
```

Comment:
should we by default be using the `coreth.params`? What if `subnet-evm` has different values, and the message we're sending doesn't interact with the primary network

Reply:
Yeah, we should make that distinction. I added a comment to https://github.com/ava-labs/awm-relayer/issues/8 to fetch the numerator config from the VM, rather than using a hardcoded value.